README.md

```diff
@@ -62,7 +62,7 @@ __Hello World__
 fn main() -> Result<(), failure::Error> {
     let mut app = tide::App::new(());
     app.at("/").get(async move |_| "Hello, world!");
-    app.serve("127.0.0.1:8000")?;
+    app.serve("127.0.0.1:8000").unwrap();
```

Comment:
Instead of `unwrap`ing the value here (which would probably cause a `panic`), would you be open to the following solution?

```rust
fn main() -> Result<(), std::io::Error> {
  let mut app = tide::App::new(());
  // ...body omitted
  Ok(app.serve("127.0.0.1:8000")?)
}
```

I've confirmed that this code works without including `failure` or other error handling crates, which was another issue in our `README` example.

Reply:
Ah! I needed to check it thoroughly. My bad :) 
Thank you for the kind explanation.